pyne/partisn.py

```diff
@@ -586,7 +561,7 @@ def _write_input(title, block01, block02, block03, block04, block05, name=None):
         for num in row:
             partisn += "{} ".format(num)
             count += 1
-            if count == 20:
+            if count == 10:
```

Comment:
When I was writing this function I wasn't exactly sure how to limit line length so 20 was a guess that worked at the time. But you seem to know how to do this with wrap(s,80). It might be worth adding the wrap(s,80) right away with this PR too.